tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -561,3 +561,70 @@ def test_positive_template_check_aap_snippet(
         assert 'systemctl enable ansible-callback' in render
         assert f'"host_config_key":"{config_key}"' in render
         assert '{"package_install": "zsh"}' in render
+
+    @pytest.mark.parametrize('module_sync_kickstart_content', [7, 8, 9], indirect=True)
+    def test_positive_template_check_remote_execution_snippet(
+        self,
+        module_sync_kickstart_content,
+        module_target_sat,
+        module_provisioning_capsule,
+        module_sca_manifest_org,
+        module_location,
+        module_default_org_view,
+        module_lce_library,
+        default_architecture,
+        default_partitiontable,
+    ):
+        """Read the remote_execution_ssh_keys snippet and verify the home directory permissions are properly set.
+
+        :id: e5212c46-d269-4bce-8e03-9d00c086e69e
+
+        :expectedresults: The rendered remote_execution_ssh_keys snippet contains correct home directory permissions.
+
+        :BZ: 2243679
+
+        :customerscenario: true
+
+        :parametrized: yes
+        """
+        macaddress = gen_mac(multicast=False)
+        remote_execution_user = gen_string('alpha')
```

Comment:
Just to optimize a few lines
```suggestion
        rex_user = gen_string('alpha')
```

Reply:
Updated